src/axom/primal/operators/squared_distance.hpp

```diff
@@ -69,32 +71,30 @@ AXOM_HOST_DEVICE inline double squared_distance(const Point<T, NDIMS>& A,
  *  given axis-aligned bounding box B.
  * \param [in] P the query point.
  * \param [in] B the axis-aligned bounding box.
- * \return the squared distance from P to the closest point on box B.
+ * \return the squared distance from P to the closest point on box \a B
+ * or std::numeric_limits<T>::max() if \a B is invalid. 
  */
 template <typename T, int NDIMS>
 AXOM_HOST_DEVICE inline double squared_distance(const Point<T, NDIMS>& P,
                                                 const BoundingBox<T, NDIMS>& B)
 {
+  using axom::utilities::clampVal;
+
+  if(!B.isValid())
```

Comment:
Bugfix -- we now account for invalid bounding boxes in `squared_distance(point, bbox)`

Reply:
Value judgement -- for invalid bounding boxes, we now return `std::numeric_limits<T>::max()` instead of NaN.
This seemed less likely to get in users way, since we don't need to always check for `isNaN`.

Thoughts?

Reply:
Ok